rest/cluster-pending-folders-delete.rst

```diff
@@ -0,0 +1,19 @@
+DELETE /rest/cluster/pending/folders
+====================================
+
+.. versionadded:: 1.18.0
+
+Remove records about a pending folder, without ignoring it in the
+configuration.  Valid values for the ``folder`` and ``device``
```

Comment:
The ignoring in config part makes sense given the history, but seems out of context like this. I'd not mention it at all, or then add a full blown explanation that you should ignore devices instead, if you don't ever want to get pending entry again.

Reply:
Applied both suggestions.